geode-apis-compatible-with-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/pubsub/AbstractPubSubIntegrationTest.java

```diff
@@ -494,6 +506,25 @@ private void runSubscribeAndPublish(int index, int minimumIterations, AtomicBool
     return mockSubscriber.getReceivedEvents();
   }
 
+  @Test
+  public void ensureOrderingWithOneSubscriberMultiplePublishes() {
+    MockSubscriber mockSubscriber = new MockSubscriber();
+    executor.submit(() -> subscriber.subscribe(mockSubscriber, "salutations"));
```

Comment:
I can understand the suggestion, but actually, I'd prefer not to. Maybe just me, but I find that constants in tests add a tiny layer of abstraction that actually detract from understanding what's going on. In this case, maybe less so since the constant is a string and would obviously be something like `SALUTATIONS` or `SALUTATIONS_CHANNEL`. But, even then it's ambiguous. Is it a regular channel or is it a pattern channel? The ambiguity is more so when the constant refers to a number, say `VALUE_TWO`. Does that mean the integer `2` or a string `"2"` or the word `two`?

Reply:
Fair enough, that makes sense.